build_projects/dotnet-cli-build/dotnet-cli-build.csproj

```diff
@@ -0,0 +1,72 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
There is an issue for migration defaults, create comments there for cleaned up properties.